lib/fog/openstack.rb

```diff
@@ -289,6 +289,7 @@ def self.retrieve_tokens_v2(options, connection_options = {})
       tenant_name = options[:openstack_tenant].to_s
       auth_token  = options[:openstack_auth_token] || options[:unscoped_token]
       uri         = options[:openstack_auth_uri]
+      omit_default_port = options[:openstack_auth_omit_default_port]
```

Comment:
indentation nit, we should have rubocop soon, could you indent all = the same? Applies to all below

@seanhandley hm, what are we missing to get rubocop running for each PR?